src/test/java/test/timeout/github1493/EnsureTimeoutsErrorProperlyTest.java

```diff
@@ -0,0 +1,31 @@
+package test.github1493;
+
+import org.testng.Assert;
+import org.testng.ITestNGListener;
+import org.testng.ITestResult;
+import org.testng.TestListenerAdapter;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import org.testng.internal.thread.ThreadTimeoutException;
+import org.testng.xml.XmlSuite;
+import test.SimpleBaseTest;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class EnsureTimeoutsErrorProperlyTest extends SimpleBaseTest {
+
+    @Test
+    public void testWithOnlyOneThread() {
```

Comment:
Yeah, I guess so. No point in adding one more new class just for a new test method. I will fix this once I go home.